packages/manager/modules/vps/src/additional-disk/upgrade/upgrade.module.js

```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+
+import additionalDiskUpgrade from '../../components/additional-disk/upgrade';
+
+import routing from './upgrade.routing';
+
+const moduleName = 'ovhManagerVpsAdditionalDiskTerminate';
```

Comment:
I think is not good module name. This should maybe:
```
const moduleName = 'ovhManagerVpsAdditionalDiskUpgrade';
```
And existe already in : packages/manager/modules/vps/src/additional-disk/terminate/terminate.module.js